llvm/include/llvm/MC/MCTargetOptions.h

```diff
@@ -92,8 +92,15 @@ class MCTargetOptions {
   std::string AsSecureLogFile;
 
   const char *Argv0 = nullptr;
```

Comment:
Would you mind please moving (and adapting) the comment that you've added here above? Both Argv0 and CommandLineArgs are only used for storing a reproducible command in the Codeview debug infos.